packages/wine.rb

```diff
@@ -81,13 +85,8 @@ def self.install
     end
     Dir.chdir 'wine64-build' do
       system 'make', "DESTDIR=#{CREW_DEST_DIR}", "DLLDIR=#{CREW_DEST_DLL_PREFIX}", 'install'
+      FileUtils.install 'wine_config_env', "#{CREW_DEST_PREFIX}/etc/env.d/wine", mode: 0o644
     end
-    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/env.d/"
-    @wine_config_env = <<~'WINE_CONFIG_EOF'
-      # Wine configuration
-      WINEPREFIX="${XDG_CONFIG_HOME}"/.wine
-    WINE_CONFIG_EOF
-    File.write("#{CREW_DEST_PREFIX}/etc/env.d/wine", @wine_config_env)
   end
 
   def self.postinstall
```

Comment:
Add message to `source ~/.bashrc` to set the envar?

Reply:
Done.